### DISCUSSION TITLE
Textareas in issue forms cannot be resized vertically

### DISCUSSION THREAD

**Discussion by mootari** [community/community#75710] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Textareas in issue forms are currently locked to 100px height and can only be resized horizontally.

The source appears to be the following CSS snippet (which is currently bundled in [this file](https://github.githubassets.com/assets/github-682d1d4acd34.css)):
~~~css
.issue-form-textarea {
    height: 100px !important;
    min-height: 100px !important
}
~~~
Spotted on https://github.com/fastify/fastify-bearer-auth/issues/new?template=bug.yml (there are two textareas further down).

**Comment by Kushalshakya** [comment]
I might be wrong correct me if but the height & min-height is doing nothing else only taking some bits of storage just use max-height and the second thing about the textarea issue. The textarea has free resizeable property (can be toggled off by resize: none) and what does it mean it cannot be resized vertically. Or is it the problem with your width?

**Reply by mootari** [reply]
@Kushalshakya did you try to resize the textareas in the link that I shared?

Element resizing via `resize` works by setting the respective style property on the element. A rule that defines the height via `!important` overrides the element's inline property.

Here is a demo:

<video src="https://github.com/community/community/assets/322014/4da8e039-34f7-4990-9d53-97553404f17e">

**Reply by Kushalshakya** [reply]
@mootari I know this that !important does override the existing style that has been applied and why don't you try to remove the height property and only keep the min-height. And I am sorry to say where is the link I am unable to find in your discussion.

**Reply by mootari** [reply]
**This is a bug report for a GitHub feature**, not a support request.
